client/src/components/TechnolgiesWeUse/TechnologiesWeUse.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
```

Comment:
Same comment on this file. I feel it should go in the Home directory since it is part of the home page.